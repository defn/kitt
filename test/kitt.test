#!/bin/bash

function main {
  kitt
}

main "$@"
