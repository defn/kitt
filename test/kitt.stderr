USAGE: kitt sub-command

sub-commands:
  up: bring up services
  down: shut down services
  restart: restart services
  recreate: delete and bring up services
  recycle: pull images and recreate services

flags:
  -h,--help:  show this help (default: false)
